ods_ci/tests/Resources/OCP.resource

```diff
@@ -333,17 +333,15 @@ Wait For Namespace To Be Active
     Log    ${value}
     Should Be Equal As Integers    ${rc}    0
 
-Is Cluster Type Self-Managed
```

Comment:
Could you maintain this keyword returning a negation of `Is Cluster Type Managed` ?

Something like:
```
    ${is_managed}=    Is Cluster Type Managed
    RETURN    not ${is_nanaged}
```

Reply:
"Is Cluster Type Managed" keyword covers both the cases for cluster type (managed and self-managed) in one single keyword, which is sufficient in my opinion.

Reply:
I can keep it `Is Cluster Type Self-Managed` if you have a strong preference

Reply:
No, no need. You are right. Thanks @ChughShilpa !